helm/conjur-config-cluster-prep/README.md

```diff
@@ -180,6 +182,27 @@ The steps are as follows:
    -----END CERTIFICATE-----
    ```
 
+   The script will display the fingerprint for the certificate and ask if
+   you trust the certificate.
+
+   For Conjur Enterprise, the fingerprint can be verified by running
+   the following command on the Conjur instance (either leader or follower,
+   depending upon the Conjur appliance URL):
+   ```
+   openssl x509 -fingerprint -noout -in <certificate file path>
+   ```
+   The certificate is located at ~conjur/etc/ssl/conjur.pem
+
+   In a Kubernetes cluster, the certificate is located at
+   /opt/conjur/etc/ssl/cert/tls.crt in the nginx container in the conjur server pod.
+   OpenSSL may need to be installed in the container.
+
+   For example:
+
+```
```

Comment:
Small nit, this codeblock needs to be indented 3 spaces since it's part of an entry in a numbered list, so it needs to match the indentation of the rest of the text in that entry.

Without the 3-space indentation, what's happening is that the *NEXT* item on the list (line #206 below) is getting numbered starting from `1`.